ui/_locales/en/messages.json

```
@@ -550,8 +550,12 @@
       "asset.label.balance": "Balance",
       "asset.label.symbol": "Name",
       "networkSelect.title": "Select Network",
-      "warning.alreadyExists.title": "Asset already exists, you will see it once your balance is over 0",
-      "warning.alreadyExists.desc": "If you still have issues seeing an asset, turn off “Hide asset balances under 2$” from Settings",
+      "warning.alreadyExists.title": "Asset already exists",
+      "warning.alreadyExists.desc": {
+        "dust": "Your balance is bellow $2, you will see it once your turn off “Hide asset balances under 2$” in Settings",
+        "noBalance": "Your balance is 0, you will see it once your balance is over $2",
```

```suggestion
        "noBalance": "Your balance is 0, you will see it once your balance is over $2.",
```

Let's also interpolate that $2 if possible, so we're set up to make this threshold configurable in the future (more or less necessary if we ever make the user's currency configurable). I'm fine with the interpolation being a constant "$2" in the component, to be clear, just want it to be interpolated so we're not having to revise i18n keys later.